### DISCUSSION TITLE
I can't see some characters in Helix while I can see them in other editors

### DISCUSSION THREAD

**Discussion by jozsefk9** [helix-editor/helix#11254] [General]
Anyone knows if there is a way to see these characters in Helix, in some similar way as I can see them in these other three editors (Vim, CudaText and ECode)?

Helix is at the upper right part of the screenshot bellow. Character I see in other editors are shown as `<feff>` in Vim; `xFEFF` in CudaText; and in ECode editor it looks like some kind of ibeam.

Get the .txt file from [here](https://filetransfer.io/data-package/tBWW9chC#link)

![20Jul2024-19^24^47](https://github.com/user-attachments/assets/c64022b9-f283-4571-9227-0ded5e017a98)

**Comment by kirawi** [comment]
You can enable rendering special characters for whitespace: https://docs.helix-editor.com/editor.html#editorwhitespace-section

**Reply by jozsefk9** [reply]
> You can enable rendering special characters for whitespace: https://docs.helix-editor.com/editor.html#editorwhitespace-section

That didn't do it. I still can't see or select anything:

![20Jul2024-22^54^15](https://github.com/user-attachments/assets/3b097ea0-7a9d-40e3-b79e-bc590a69f52d)

**Reply by kirawi** [reply]
Alright, I see the issue. We don't support a whitespace character for that as well as some other whitespace characters. I'll create an issue to track these.

**Reply by kirawi** [reply]
https://github.com/helix-editor/helix/issues/11255
